### PR TITLE
Fix up pip install on windows and py2exe with recent setuptools

### PR DESCRIPTION
**Pull request by lambacck**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
 import os
 import re
 import sys
+from setuptools import setup
```
Comment (unho) [inline setup.py]: Please put this a few lines below so imports are ordered alphabetically:

```python
from distutils.sysconfig import get_python_lib
from os.path import dirname, isfile, join
from setuptools import setup
```

File: setup.py [hunk 2]
```diff
@@ -456,6 +456,11 @@ def dosetup(name, version, packages, datafiles, scripts, ext_modules=[]):
           ext_modules=ext_modules,
           cmdclass=cmdclass,
           install_requires=parse_requirements('requirements/required.txt'),
+          entry_points={
+              'console_scripts': [
+                  os.path.basename(x) + ' = ' + x.replace(os.path.sep, '.') + ':main' for x in translatescripts
```
Comment (unho) [inline setup.py]: Please break this line:

```python
                  os.path.basename(x) + ' = ' + x.replace(os.path.sep, '.') + ':main'
                  for x in translatescripts
```

**Comment by claudep** [comment]
Isn't this patch conflicting with #3362?

File: setup.py (hunk 2 above)
Comment (unho) [inline setup.py]: Shouldn't `translatebashscripts` be included here as well?

**Comment by unho** [comment]
> Isn't this patch conflicting with #3362?

I think so. #3662 looks like it is a nicer cleanup.

**Comment by lambacck** [comment]
#3662 is a nicer cleanup but it means you loose the inslaller and py2exe. That means on Windows users need to install python. That's totally fine for my use case where I'm using it with another python application.  Right now there is no way to install the tool on Windows without knowing to go to download version 1.9 from sourceforge.

**Comment by unho** [comment]
Closed in favor of #3662.

**Pull request closed without merging**